### PR TITLE
bpo-32360: Remove OrderedDict reference as dicts are ordered now

### PR DESCRIPTION
**Pull request by srinivasreddy**
https://bugs.python.org/issue32360

### CODE REVIEW

File: Lib/unittest/util.py [hunk 1]
```diff
@@ -153,17 +153,17 @@ def _count_diff_all_purpose(actual, expected):
         result.append(diff)
     return result
 
-def _ordered_count(iterable):
-    'Return dict of element counts, in the order they were first seen'
-    c = OrderedDict()
+def _count(iterable):
+    'Return dict of element counts'
```
Comment (merwok) [inline Lib/unittest/util.py]: In my opinion changing the function name and doc is not an improvement.

**Comment by srinivasreddy** [comment]
@asvetlov  Just a question. Tagging someone who can review and merge is fine here? If so could you please look into this PR? 😄

File: Misc/NEWS.d/next/Documentation/2017-12-24-17-29-37.bpo-32360.eZe-ID.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Remove references of OrderedDict in module `unittest`
```
Comment (merwok) [inline Misc/NEWS.d/next/Documentation/2017-12-24-17-29-37.bpo-32360.eZe-ID.rst]: I find «reference» a bit vague, what do you think about using «import» instead?

File: Lib/unittest/util.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 """Various utility functions."""
 
-from collections import namedtuple, OrderedDict
+from collections import namedtuple
```
Comment (merwok) [inline Lib/unittest/util.py]: The collections module ends up loaded anyway, so removing OrderedDict doesn’t really change anything for this module.  @rhettinger would you mind giving your opinion for this PR?

File: Misc/NEWS.d/next/Documentation/2017-12-24-17-29-37.bpo-32360.eZe-ID.rst (hunk 2 above)
Comment (methane) [reply Misc/NEWS.d/next/Documentation/2017-12-24-17-29-37.bpo-32360.eZe-ID.rst]: I don't think NEWS entry is required for this.
You can use "skip news" label instead

**Review by methane — CHANGES_REQUESTED** [review]
Why don't you use collections.Counter instead?

**Comment by srinivasreddy** [comment]
@methane  Done. Thanks for the review.

**Comment by srinivasreddy** [comment]
@methane  Please add skip-news label too.!!

**Review by methane — APPROVED** [review]

**Pull request merged**